### PR TITLE
Update tutorial for solution deployment

### PR DESCRIPTION
**Pull request by kumaripriyankasingh**
Adding below in tutorial:
* Fixes for creating cloud storage bucket
* Details for opening workspace in both legacy and preview cloud shell editor
* Steps for deleting the deployed solution
* Minor rewordings

### CODE REVIEW

File: tutorial.md [hunk 1]
```diff
@@ -7,13 +7,24 @@
...
 
 To avoid conflicts, only one user should modify and deploy a solution in a single GCP project.
 
 ## Open cloned repository as workspace
 
-Open the directory where the repository is cloned as a workspace in the editor.
+Open the directory where the repository is cloned as a workspace in the editor, follow the steps based on whether you are using the Cloud Shell Editor in Preview Mode or Legacy Mode.
+
+---
+**Cloud Shell Editor in Preview Mode**
```
Comment (singhalbhaskar) [inline tutorial.md]: Is it preview mode? Or just New Cloud Shell Editor?

File: tutorial.md [hunk 2]
```diff
@@ -7,13 +7,24 @@
...
+
+* Go the hamburger icon located in the top left corner of the editor.
+* Go to the `File` Menu.
+* Select `Open Folder`.
+* Choose the directory where the repository has been cloned. This directory is the current directory in the cloud shell terminal.
+
+**Cloud Shell Editor in Legacy Mode**
```
Comment (singhalbhaskar) [inline tutorial.md]: Legacy Cloud Shell Editor

**Review by singhalbhaskar — COMMENTED** [review]
Order of Legacy and New should be reversed.
Legacy Cloud Shell Editor
New Cloud Shell Editor

File: tutorial.md [hunk 3]
```diff
@@ -170,5 +195,14 @@ git remote set-url origin [git-repo-url]
 ```
 
 Review the modified files, commit and push to your remote repository branch.
+
+## Delete the deployed solution
+
+Optional: Use one of the below options in case you want to delete the deployed solution
+
+* Go to [JSS deployment page](https://console.cloud.google.com/products/solutions/deployments?pageState=(%22deployments%22:(%22f%22:%22%255B%257B_22k_22_3A_22Labels_22_2C_22t_22_3A13_2C_22v_22_3A_22_5C_22modification-reason%2520_3A%2520make-it-mine_5C_22_22_2C_22s_22_3Atrue_2C_22i_22_3A_22deployment.labels_22%257D%255D%22))).
```
Comment (singhalbhaskar) [inline tutorial.md]: I think the page is called Solution deployments

File: tutorial.md (hunk 2 above)
Comment (kumaripriyankasingh) [reply tutorial.md]: Done.

File: tutorial.md (hunk 1 above)
Comment (kumaripriyankasingh) [reply tutorial.md]: Updated to "New Cloud Shell Editor"

File: tutorial.md (hunk 3 above)
Comment (kumaripriyankasingh) [reply tutorial.md]: Sure. Done.

**Review by singhalbhaskar — APPROVED** [review]

**Pull request merged**